### PR TITLE
Scenes: Panel repeater change

### PR DESCRIPTION
**Pull request by torkelo**
Scenes PR: https://github.com/grafana/scenes/pull/1

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -21990,7 +21989,7 @@ __metadata:
     "@grafana/lezer-logql": 0.1.1
     "@grafana/monaco-logql": ^0.0.6
     "@grafana/runtime": "workspace:*"
-    "@grafana/scenes": 0.0.4
+    "@grafana/scenes": "portal:/Users/torkelodegaard/dev/scenes"
```
Comment (torkelo) [inline yarn.lock:21992]: hm.. @dprokop wonder how we should handle this workflow (PR in main and PR in scenes), think we will run into this every day coming months. 

can we pin scenes to canary in main?

Comment (torkelo) [reply yarn.lock:21992]: but that might lead to a lot build failures until the OSS PR is merged (but we could maybe merge them at the same time?) but we would have to change the canary version whenever a release branch of happens.

Comment (dprokop) [reply yarn.lock:21992]: Yes, but we need to enable automated publishing in @grafana/scenes first.

**Review by dprokop — APPROVED** [review]
Updated scenes to use `latest` version.

**Pull request merged**